packages/base/tests/functional/basics/UnitTest.lua

```diff
@@ -55,8 +55,8 @@ return{
 		unitTest:assertEquals(1, 1.1, 0.15)
 		unitTest:assertEquals("abc", "abd", 1)
 
-		local expected = [[string [biomassa-manaus.asc] ]]
 		local actual = [[string [/home/jenkins/Documents/ba1c13592dcf65f3d0b2929f8eff266c4e622470/install/bin/packages/terralib/data/biomassa-manaus.asc] ]]
+		local expected = [[string [biomassa-manaus.asc] ]]
```

Comment:
It is better to use `"` to describe strings that have only one line. Use `[[ ]]` only for strings with multiple lines.